src/commands/BootstrapCommand.js

```diff
@@ -204,8 +204,8 @@ export default class BootstrapCommand extends Command {
         .forEach((dependency) => {
           // get Package of dependency
           const dependencyPackage = this.packageGraph.get(dependency).package;
-          // get path to dependency and its package.json
-          const dependencyLocation = dependencyPackage.location;
+          // get path to dependency and its scope
+          const { location: dependencyLocation, scope: dependencyScope } = dependencyPackage;
```

Comment:
with scopes involved, the destination folder is actually `node_modules/<@scope>/<name>`, not just `node_modules/<name>`. i wanted a way to extract the scope and use that for building the path.